src/boot/ksdk1.1.0/devIS25xP.c

```diff
@@ -196,20 +196,15 @@ programSingleIS25xPWithoutOffsetUpdate(uint16_t* pageNumber_p, uint8_t* pageOffs
 		status = programPageIS25xP(*pageNumber_p, *pageOffset_p, gFlashWriteLimit, buf + (i * gFlashWriteLimit));
 		*pageOffset_p += gFlashWriteLimit;
 
-		if (status != kWarpStatusOK)
-		{
-			warpPrint("\r\n\tError: programPageIS25xP failed");
-			return status;
-		}
+		// if (status != kWarpStatusOK)
+		// {
+		// 	warpPrint("\r\n\tError: programPageIS25xP failed");
+		// 	return status;
+		// }
```

Comment:
Why remove the error return?

Reply:
It's because the binary is taking too much memory. It's a kind of **region `m_text' overflow** error. So, I am forced to temporarily comment out some error returns?
Here is an example of the error after compiling the code:
```
/Users/hamid/bare-metal-arm/gcc-arm-none-eabi-4_8-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld: region `m_text' overflowed by 472 bytes
```
It's related to the following issue link:
[](https://github.com/physical-computation/Warp-firmware/issues/130)

Reply:
What if you try to compile without the `warpPrint` but including the if-return?